### DISCUSSION TITLE
Vim/Neovim-like undofile.

### DISCUSSION THREAD

**Discussion by Chirikumbrah** [helix-editor/helix#9648] [General] [answered]
Is it possible or maybe planned to implement a feature like `undofile` vim option?
It might be really convenient when you saved the file and exited helix. But then you can open the file again and undo the changes by pressing `u` key :)

**Comment by kirawi** [comment, marked as answer]
https://github.com/helix-editor/helix/pull/9154

**Comment by Chirikumbrah** [comment]
> https://github.com/helix-editor/helix/pull/9154

Wow! It's incredible. Thank you, bro😁
It would be awesome to see this feature in the next release)
